tests/sentry/feedback/test_feedback_ingest.py

```diff
@@ -0,0 +1,53 @@
+from django.urls import reverse
+
+from sentry.testutils.cases import MonitorIngestTestCase
+
+test_data = {
+    "contexts": {},
+    "tags": {
+        "sentry_version": "23.9.0.dev0",
+    },
+    "timestamp": 1694039635.9195,
+    "transaction": "/replays/",
+    "type": "transaction",
+    "transaction_info": {"source": "route"},
+    "platform": "javascript",
+    "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+    "environment": "prod",
+    "release": "frontend@40f88cd929122ac73749cc48f0ddb9aa223449ff",
+    "sdk": {"name": "sentry.javascript.react", "version": "7.66.0-alpha.0"},
+    "user": {
+        "ip_address": "72.164.175.154",
+        "email": "josh.ferge@sentry.io",
+        "id": 880461,
+        "isStaff": False,
+        "name": "Josh Ferge",
+    },
+    "request": {
+        "url": "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d",
+        "headers": {
+            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"
+        },
+    },
+}
+
+
+class FeedbackIngestTest(MonitorIngestTestCase):
+    endpoint = "sentry-api-0-feedback-ingest"
+
+    def test_save_with_feedback(self):
+        path = reverse(self.endpoint)
+        response = self.client.post(path, data=test_data, **self.dsn_auth_headers)
+        assert response.status_code == 201, response.content
+
+    def test_get_feedback_list(self):
+        path = reverse(self.endpoint)
+        response = self.client.post(
+            path,
+            data=test_data,
+            **self.dsn_auth_headers,
+        )
+        response = self.client.get(path, **self.dsn_auth_headers)
+        assert response.status_code == 200, response.content
+        assert len(response.data) == 1
+        assert response.data[0]["data"]["type"] == "transaction"
```

Comment:
Now is a really great time to add lots and lots of test coverage.  It's better to find problems now rather than next week when we're trying to integrate with the front-end.

Let's assert the structure of the response.  Let's assert any HTTP parameters you've added are functional.  Let's assert the variety of valid messages that can be ingested by this endpoint.  Assert authorization failures.  Assert disabled features, invalid orgs, etc.  We want to have the clearest picture possible of how this will behave in production.

Reply:
@michellewzhang If any of this is overwhelming or if you're not sure how to test these things, ping me on slack and we can pair.